qiskit/qobj/_qobj.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Models for QObj and its related components."""
+
+from types import SimpleNamespace
+
+from ._utils import QObjType
+
+
+class QObjStructure(SimpleNamespace):
+    """
+    General QObj structure.
+    """
+    REQUIRED_ARGS = ()
```

Comment:
There are indeed several options for the next steps in this regard and some related issues (how to ideally build the class from the schema, etc) that need discussion; and I would indeed like to reduce the amount of repetition in further updates.

In this case, I went with a conservative approach giving priority to the goal of having the class working and keeping the `REQUIRED_ARGS` explicit for clarity, and be able split the remaining work (for example, that change would be working on the internals) afterwards to avoid blocking - would that make sense?